src/components/text-input/index.md.njk

```diff
@@ -21,9 +21,15 @@ Do not use the text input component if you need to let users enter longer answer
 
 ## How it works
 
-All text inputs must have visible labels; placeholder text is not an acceptable replacement for a label as it vanishes when users start typing.
+All text inputs must have labels, and in most cases the label should be visible.
 
-Labels should be aligned above the text input they refer to. They should be short, direct and written in sentence case. Do not use colons at the end of labels.
+You should align labels above the text input they refer to. They should be short, direct and written in sentence case. Do not use colons at the end of labels.
+
+Do not use placeholder text in place of a label, or for hints or examples, as:
```

Comment:
Any idea if 'placeholder text' is the best term for this? I'll do a check when I can.

Reply:
Not sure, although we already use the term in the text input and the [textarea](https://design-system.service.gov.uk/components/textarea/#:~:text=Placeholder%20text%20is%20not%20a%20suitable%20substitute%20for%20a%20label%2C%20as%20it%20disappears%20when%20users%20click%20inside%20the%20textarea.) pieces of guidance.

Reply:
Hey both! I can confirm that 'placeholder text' is the term used in web development. Here's an example of its use in some W3C Web Accessibility Initiative documentation: https://www.w3.org/WAI/tutorials/forms/instructions/#placeholder-text

Reply:
Nice, resolved.